src/test/operators/aggregate_test.cpp

```diff
@@ -125,6 +125,9 @@ TEST_F(OperatorsAggregateTest, OperatorName) {
       std::vector<ColumnID>{ColumnID{0}});
 
   EXPECT_EQ(aggregate->name(), "Aggregate");
+
+  bool did_markus_already_write_tests_with_three_aggregates = false;
```

Comment:
+1

Reply:
On purpose - I wanted to stop this from being merged but didn't feel like writing the tests before we decide that we want the changes to the Aggregate op.